Properly enable the xglk build if the XGLK port option has been turned on.

--- source/Makefile.orig
+++ source/Makefile
@@ -73,7 +73,7 @@
 # If your make can't handle sinclude or -include, and you're not building
 # for Glk, comment out the include line.
 GLKPATH	= $(GLKDIR)/$(GLK)
-sinclude $(GLKPATH)/Make.$(GLK)
+.include "$(LOCALBASE)/share/xglk/Make.$(GLK)"
 GLKLIBS	= -L$(GLKPATH) $(GLKLIB) $(LINKLIBS)
 
 # Always use the Bourne shell.
@@ -134,7 +134,7 @@
 
 os_glk.o: os_glk.c $(HEADERS)
 	$(CC) $(GLKARCH) $(CFLAGS) -Wno-write-strings			\
-		-I $(GLKDIR)/$(GLK) -c -o $@ $<
+		-DLINUX_GRAPHICS -I${LOCALBASE}/include/xglk -c -o $@ $<
 
 # Option to build an IFP plugin version of SCARE.
 scare_plugin.c: scare.hdr
